src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/GetTaskByIdControllerTest.java

```diff
@@ -98,7 +105,13 @@ void should_return_a_403_when_restricted_role_is_given() throws Exception {
 
         mockServices.mockUserInfo();
 
-        final List<String> roleNames = asList("tribunal-caseworker");
+        when(launchDarklyFeatureFlagProvider.getBooleanValue(
+                FeatureFlag.RELEASE_2_CANCELLATION_COMPLETION_FEATURE,
+                IDAM_USER_ID, IDAM_USER_EMAIL
+            )
+        ).thenReturn(false);
```

Comment:
we are mocking the response for RELEASE_2_CANCELLATION_COMPLETION_FEATURE inside the GetTaskByIdcontroller this feels like a typo can we review

Reply:
removed